tests/test_checkout.py

```diff
@@ -143,6 +160,23 @@ def test(self):
         self.assertFalse(os.path.exists(fname))
 
 
+class TestCheckoutSelectiveRemove(CheckoutBase):
+    def test(self):
+        # Use copy to test for changes in the inodes
+        main(['config', 'cache.type', 'copy'])
+
+        stage = self.dvc.add('data_dir')[0]
+        staged_files = self.outs_info(stage)
+
+        os.remove(staged_files[0].path)
+        main(['checkout', 'data_dir.dvc'])
```

Comment:
```suggestion
        ret = main(['checkout', stage.relpath])
        self.assertEqual(ret, 0)
```

Reply:
This one too, there's already a test for the `checkout` command executed from the CLI

Reply:
Same here. Could we add this as well please?